crates/rari-doc/src/helpers/summary_hack.rs

```diff
@@ -46,3 +47,8 @@ pub fn strip_paragraph_unchecked(input: &str) -> &str {
 
     out
 }
+
+pub fn text_content(html_str: &str) -> String {
```

Comment:
Nit: Maybe `html_to_text()`

```suggestion
pub fn html_to_text(html_str: &str) -> String {
```

Reply:
it comes from https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent